rotkehlchen/types.py

```diff
@@ -837,6 +838,57 @@ def __str__(self) -> str:
         return f'Addressbook entry with name "{self.name}", address "{self.address}" and blockchain {str(self.blockchain) if self.blockchain is not None else None}'  # noqa: E501
 
 
+class LocationAssetMappingDeleteEntry(NamedTuple):
```

Comment:
This should inherit from `LocationAssetMappingUpdateEntry`. And probably would need to use dataclasses